gapic-generator-java/src/test/resources/showcase_v1beta1.yaml

```diff
@@ -0,0 +1,93 @@
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+type: google.api.Service
+config_version: 3
+name: showcase.googleapis.com
+title: Client Libraries Showcase API
+
+apis:
+  - name: google.showcase.v1beta1.Compliance
+  - name: google.showcase.v1beta1.Echo
+  - name: google.showcase.v1beta1.Identity
+  - name: google.showcase.v1beta1.Messaging
+  - name: google.showcase.v1beta1.SequenceService
+  - name: google.showcase.v1beta1.Testing
+  # Mix-in services
+  - name: 'google.cloud.location.Locations'
+  - name: 'google.iam.v1.IAMPolicy'
+  - name: 'google.longrunning.Operations'
+
+documentation:
+  summary: |-
+    Showcase represents both a model API and an integration testing surface for
+    client library generator consumption.
+
+backend:
+  rules:
+    - selector: 'google.cloud.location.Locations.*'
+      deadline: 60.0
+    - selector: 'google.iam.v1.IAMPolicy.*'
+      deadline: 60.0
+    - selector: 'google.longrunning.Operations.*'
+      deadline: 60.0
+
+http:
+  rules:
+    - selector: google.cloud.location.Locations.ListLocations
+      get:  '/v1beta1/{name=projects/*}/locations'
+    - selector: google.cloud.location.Locations.GetLocation
+      get:  '/v1beta1/{name=projects/*/locations/*}'
+    - selector: google.iam.v1.IAMPolicy.SetIamPolicy
+      post: '/v1beta1/{resource=users/*}:setIamPolicy'
+      body: '*'
+      additional_bindings:
+        - post: '/v1beta1/{resource=rooms/*}:setIamPolicy'
+          body: '*'
+        - post: '/v1beta1/{resource=rooms/*/blurbs/*}:setIamPolicy'
+          body: '*'
+        - post: '/v1beta1/{resource=sequences/*}:setIamPolicy'
+          body: '*'
+    - selector: google.iam.v1.IAMPolicy.GetIamPolicy
+      get: '/v1beta1/{resource=users/*}:getIamPolicy'
+      additional_bindings:
+        - get: '/v1beta1/{resource=rooms/*}:getIamPolicy'
+        - get: '/v1beta1/{resource=rooms/*/blurbs/*}:getIamPolicy'
+        - get: '/v1beta1/{resource=sequences/*}:getIamPolicy'
+    - selector: google.iam.v1.IAMPolicy.TestIamPermissions
+      post: '/v1beta1/{resource=users/*}:testIamPermissions'
+      body: '*'
+      additional_bindings:
+        - post: '/v1beta1/{resource=rooms/*}:testIamPermissions'
+          body: '*'
+        - post: '/v1beta1/{resource=rooms/*/blurbs/*}:testIamPermissions'
+          body: '*'
+        - post: '/v1beta1/{resource=sequences/*}:testIamPermissions'
+          body: '*'
+    - selector: google.longrunning.Operations.ListOperations
+      get: '/v1beta1/operations'
+      additional_bindings:
+        - get: '/v1/operations'
```

Comment:
Is it possible to have multiple additional bindings here? If yes, can we add more here to cover the test case?

Reply:
In addition, do you know how `additional_bindings` are used in LROs or is it even used? If not, we probably can get rid of the duplicated logics.

Reply:
Yep, I'll add more bindings for this. Thanks!

Reply:
> In addition, do you know how `additional_bindings` are used in LROs or is it even used? If not, we probably can get rid of the duplicated logics.

I've actually just been doing a bit of digging into as I've been running into some showcase issues with `additional_bindings`. For httpjson, I don't think there currently is any logic to handles checking other bindings once the first one doesn't match: https://github.com/googleapis/gapic-generator-java/blob/c23f981e2ac3c573bed51e725dc7061551179400/gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/HttpRequestRunnable.java#L184

I don't see any logic that handles checking the other paths/ additional_bindings which I think is a bug. For grpc, I'll need to do some more digging.

Reply:
Filed a bug here: https://github.com/googleapis/gapic-generator-java/issues/1440

Reply:
Thanks Lawrence, that bug looks generic for all `additional_bindings`, anything special for the OperationClient? Or as long as it's REST, there is no differences between OperationClient and other clients?

Reply:
Yeah, the bug is generic for all `additional_bindings` and it's a very specific edge case. 

There is nothing special for OperationsClient, but I just added multiple `additional_bindings` to the Operations Client for showcase.